src/mumble/AudioWizard.h

```diff
@@ -46,7 +46,7 @@ class AudioWizard : public QWizard, public Ui::AudioWizard {
 	int iMaxPeak;
 	int iTicks;
 
-	void restartAudio();
+	void restartAudio(bool);
```

Comment:
I prefer headers also naming their parameters as this makes it easier to get a grasp on what they're supposed to be used for